src/main/webapp/js/components/widgets/connectivity/Connectivity.js

```diff
@@ -33,7 +34,7 @@ define(function (require) {
             layout: "matrix", //[matrix, force, hive, chord]
             nodeType: function (node) {
                 if (node instanceof Instance) {
-                    return node.getType().getId();
+                    return node.getParent().getId();
```

Comment:
@mattearnshaw while every instance will have a type it's not sure it will have a parent I think, what assumption are you using here?

Reply:
since calls to `nodeType` (L172) are on children only. this change was requested by @pgleeson so the legend displays the population id rather than cell type, since in certain models the same cell type is used in multiple populations and so there are duplicates in the legend.